c++/src/capnp/layout.h

```diff
@@ -1261,6 +1261,13 @@ inline OrphanBuilder& OrphanBuilder::operator=(OrphanBuilder&& other) {
   return *this;
 }
 
+#define OUT_OF_BOUNDS_ERROR_DETAIL \
```

Comment:
Let's put this in `layout.c++` instead of the header. All Cap'n Proto users inlcude this header so this pollutes the macro space, but this is only legitimately used in `layout.c++` anyway.